src/C-DEngine/C-DCommunication/cde_TheCommCore.cs

```diff
@@ -511,7 +510,6 @@ internal static bool ProcessISBRequest(TheRequestData pRequestData)
                 if (tRealPage.EndsWith(".ashx"))
                 {
                     tRealPage = tRealPage.Substring(0, tRealPage.Length - 5);
-                    IsValidISB = true;
```

Comment:
This is not correct. IsValidISB is only true if "ParseISBPath" return bool. 
There could be other .ashx pages that are not valid ISBPaths

Reply:
I don't fully understand your comment.
I removed `IsValidISB = true` because it seems to be wrong for that case for me. Should I better revert that change?